<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElabWindow</class>
 <widget class="QMainWindow" name="ElabWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiKinect TCPIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_Viewer">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="CloudViewer" name="myCloudViewer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_PointCloud">
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Point Clouds</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="2" column="0">
      <widget class="QPushButton" name="EstimateNormals">
       <property name="text">
        <string>Estimate Normals</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>path [..]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Line Alignement</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QPushButton" name="LineNext">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="LineAlign">
          <property name="text">
           <string>Line Align</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="lineindex_new">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="LinePrev">
          <property name="text">
           <string>Prev.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="lineindex_ref"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="LignAlignAll">
          <property name="text">
           <string>Align All Ligns</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>LineAlign</zorder>
       <zorder>lineindex_new</zorder>
       <zorder>LinePrev</zorder>
       <zorder>LineNext</zorder>
       <zorder>lineindex_ref</zorder>
       <zorder>LignAlignAll</zorder>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="IDlistWidget">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>100</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Radius Outliers</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="RemoveOutliers">
          <property name="text">
           <string>remove outliers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Refine All Point Cloud</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>LUM</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>BackBone Alignment</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QComboBox" name="IDlistcomboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="BackBoneAlign">
          <property name="text">
           <string>BackBone Align</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenAcq"/>
   <addaction name="actionShowPC"/>
   <addaction name="actionSavePointCloudAs"/>
   <addaction name="actionExportReconstruction"/>
  </widget>
  <action name="actionOpenAcq">
   <property name="text">
    <string>openAcq</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionShowPC">
   <property name="text">
    <string>ShowPC</string>
   </property>
  </action>
  <action name="actionSavePointCloudAs">
   <property name="text">
    <string>SavePointCloudAs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExportReconstruction">
   <property name="text">
    <string>ExportReconstruction</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CloudViewer</class>
   <extends>QWidget</extends>
   <header>CloudViewer.h</header>
   <container>1</container>
   <slots>
    <slot>showPC(PointCloudT::Ptr)</slot>
    <slot>setPointCloudPose(std::string,Transform)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>LinePrev</sender>
   <signal>clicked()</signal>
   <receiver>lineindex_new</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineNext</sender>
   <signal>clicked()</signal>
   <receiver>lineindex_new</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LinePrev</sender>
   <signal>clicked()</signal>
   <receiver>lineindex_ref</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineNext</sender>
   <signal>clicked()</signal>
   <receiver>lineindex_ref</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenAllDevices()</slot>
  <slot>CloseAllDevices()</slot>
  <slot>GrabAllDevices()</slot>
  <slot>TransformationChanged(TransformT)</slot>
 </slots>
</ui>
